RobotsWantedLeague/appsettings.json

```diff
@@ -5,5 +5,10 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
+  "RobotsDatabase": {
```

Comment:
This is the configuration needed to run the program using a mongodb instance running locally on mongodb's default port.